### ISSUE TITLE
[Editor] Dialogs containing input elements fail accessibility rules

### ISSUE THREAD

**Issue by slavenai** [telerik/kendo-angular#3819]
**Describe the bug**
Dialogs containing input elements fail accessibility evaluation with the **Missing form label** error (Tested with WAVE):

![image](https://user-images.githubusercontent.com/67437411/196930366-e820e306-96c1-4333-b988-d7a7a38577cf.png)

**To Reproduce**
https://stackblitz.com/edit/angular-nsuq1y-yowqpo?file=src%2Fapp%2Fapp.component.ts

**Comment by Raisolution** [comment]
Fixed in the latest version of the Editor package.
